src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java

```diff
@@ -279,7 +291,11 @@ public final void visitToken(DetailAST blockCommentNode) {
             }
 
             if (result.getParseErrorMessage() == null) {
-                processTree(result.getTree());
+                final DetailNode javadocTree = result.getTree();
+                if (doesCheckAcceptJavadocWithUnclosedHtmlTag()
+                        || !JavadocUtils.hasUnclosedTag(javadocTree)) {
```

Comment:
@romani 

Yes, that sounds nice. Also, right now checks would be forced to skip javadoc if javadoc node `hasUnclosedTag`, they can't _" simply remove all tags and keep only their text."_ right ?